frontend/docsettings.lua

```diff
@@ -294,9 +294,22 @@ function DocSettings:flush(data, no_custom_metadata)
 end
 
 --- Purges (removes) sidecar directory.
-function DocSettings:purge(sidecar_to_keep)
+function DocSettings:purge(sidecar_to_keep, data_to_purge)
+    local custom_cover_file, custom_metadata_file
+    if sidecar_to_keep == nil then
```

Comment:
Remind me when/for what this `sidecar_to_keep` is used?

Reply:
https://github.com/koreader/koreader/blob/ed2ea6803f4b9be6b065f43590229dea4fe51426/frontend/docsettings.lua#L289
In `:flush`, to keep the actual sidecar file only.